data/transactions/logic/debugger.go

```diff
@@ -192,10 +200,32 @@ func valueDeltaToValueDelta(vd *basics.ValueDelta) basics.ValueDelta {
 	}
 }
 
+// parseCallStack initializes an array of CallFrame objects from the raw
+// callstack.
+func (d *DebugState) parseCallstack(callstack []int) []CallFrame {
+	callFrames := make([]CallFrame, 0)
+	lines := strings.Split(d.Disassembly, "\n")
+	for _, pc := range callstack {
+		// The callsub is pc - 3 from the callstack pc
+		callsubLineNum := d.PCToLine(pc - 3)
```

Comment:
I did a bit of hardcoding here (`callsub` saves `pc+3` to the callstack in `eval.go`) and added a "soft" check to see if the line is a `callsub` later on so we can try to parse the label.